src/commons/controlBar/ControlBarShareButton.tsx

```diff
@@ -57,7 +80,7 @@ export class ControlBarShareButton extends React.PureComponent<ControlBarShareBu
         </div>
       ) : (
         <>
-          {!this.props.shortURL || this.props.shortURL === 'ERROR' ? (
+          {!this.state.isSuccess || this.props.shortURL === 'ERROR' ? (
```

Comment:
Why was the old check removed?

Reply:
now we don't use url shortener to generate shared links, isSuccess represents backend has successfully receive the code, so I think it would be better to use this to decide share link is successfully created